src/Elastic.Apm.StartupHook.Loader/Loader.cs

```diff
@@ -20,7 +20,7 @@
 namespace Elastic.Apm.StartupHook.Loader
 {
 	/// <summary>
-	/// Loads the agent assemblies, its dependent assemblies and starts it
+	/// Starts the agent
 	/// </summary>
 	internal class Loader

```

Comment:
With this change the name of the type and the whole project isn't very accurate - it's not really loading things anymore, it only subscribes. It's not a big deal, and I don't have super strong opinion, but wanted to note this. Maybe we could consider renaming.

Reply:
Loading the loader assembly and invoking the `Initialize()` method causes the Elastic.Apm.* assemblies it references to be loaded, though this is implicit now rather than being explicitly loaded. Thoughts on a better name?

Reply:
> Thoughts on a better name?

Not really - and you have a good point, it still triggers the agent dlls to be loaded. Let's leave it for now.